src/draw/DefaultDrawer.js

```diff
@@ -687,6 +733,48 @@ class DefaultDrawer {
       .target(() => targetAnchor);
   }
 
+  /**
+   * Compute a component's height and store it in it's drawOptions.
```

Comment:
Compute the component's height then store it in its drawOptions.

Reply:
done